Azure/sonic-buildimage
# [bgpcfgd]: Convert bgpcfgd and bgpmon to python3

Convert bgpmon to python3
Fix some issues in bgpmon



**- Why I did it**
Python2 is end of support

**- How I did it**
1. Used 2to3 utility
2. Manual editing to fix test failures
3. Run on the dut and fix the issues in the sources

**- How to verify it**
Build an image and verify that bgpcfgd works

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
